src/TramLite.js

```diff
@@ -194,18 +194,18 @@ class TramLite {
 	 * {@link https://tram-one.io/tram-lite/#appendShadowRootProcessor Read the full docs here.}
 	 * @param {string} matcher
 	 * @param {{ connect: function }} componentClass
-	 * @param {(node: Node) => boolean} [rootNodeTest=() => true]
+	 * @param {ShadowRoot} [shadowRoot=ShadowRoot.prototype]
```

Comment:
The major change here is the update to `appendShadowRootProcessor`, which for a short while supported a `rootNodeTest` - this was never documented though, so not considering this a breaking change.

Instead, we are supporting a `shadowRoot` parameter, which lets people focus the processor on a single component (useful when we don't want to update the behavior of all components everywhere).